readme/uk-UA.md

```
@@ -109,14 +109,14 @@ export default async function (): Promise<NextConfig> {
 
 ### Що міститься в цьому репозиторії?
 
-| Інструмент   | Коротко                                                                              | Документація                            |
-| ------------ | ------------------------------------------------------------------------------------ | --------------------------------------- |
-| **Compiler** | Локалізація React під час збірки                                                     | [/compiler](https://lingo.dev/compiler) |
-| **CLI**      | Локалізація веб- та мобільних додатків однією командою: JSON, YAML, markdown та інше | [/cli](https://lingo.dev/cli)           |
-| **CI/CD**    | Автоматичний коміт перекладів при кожному push + створення pull request за потреби   | [/ci](https://lingo.dev/ci)             |
-| **SDK**      | Переклад у реальному часі для контенту, створеного користувачами                     | [/sdk](https://lingo.dev/sdk)           |
+| Інструмент   | Коротко                                                                                     | Документація                            |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe has extra space to the left for style "compact"

(MD060, table-column-style)